lib/bridge/github/contents.rb

```diff
@@ -16,7 +16,8 @@ def issue_template
 
     def issue_template_content
       begin
-        template = Base64.decode64(issue_template['content'])
+        template = issue_template
+        template = Base64.decode64(template['content']) if template
```

Comment:
Not a fan of reusing `template` for a completely different thing. Suggest either:

1. Rename second use to `content`
2. Immediately `return` instead

Thoughts?